src/components/FoodSearchComponent/FoodSearchComponent.jsx

```diff
@@ -18,8 +18,8 @@ class FoodSearchComponent extends Component {
     this.props.dispatchFoodSearch(this.state);
   }
 
-  searchKeyword(e) {
-    this.setState({ data: e.target.value });
+  async searchKeyword(e){
+    await this.setState({ data: e.target.value });
```

Comment:
I'm curious about how this works.  I've never seen anything like it. (0o0)

Reply:
That was for the live search bar thing. Its there so that the code below it doesn't fire until state has been set to the correct data.